### DISCUSSION TITLE
exportGithubIssues task ?

### DISCUSSION THREAD

**Discussion by brucellino** [docToolchain/docToolchain#1407] [Q&A]
Hello!

TL;DR - what if we track issues in Github instead of Jira?

I can't see from the issues or discussions here whether this has been discussed before, or what the thinking is, so forgive me please if I have missed it. I would like to know whether there is any desire to add a task to export Github Issues analogous to the way that Jira issues are exported.

The [Java](https://docs.github.com/en/rest/using-the-rest-api/libraries-for-the-rest-api?apiVersion=2022-11-28#java) libraries for the REST API don't seem to be all that great.

I'm guessing that the solution to my need is to basically implement a custom task?

**Comment by rdmueller** [comment]
you are the first one to discuss this. I guess the Jira feature is rarely used. The original idea was to add issue links to docs which basically say "yes, the docs are not perfect, but we are working on it - check the state of this issue"

We now have the "create an issue" feature in the top right corner which already helps a bit. Microsoft managed to implement a feature in their docs, where I can see how many issues have been created for a certain page.

Maybe we find a github issue view which we can embed in the right column?

Found something:
https://stackoverflow.com/questions/33575850/is-there-any-way-to-embed-the-github-issue-tracker-in-a-webpage

**Comment by rdmueller** [comment]
Just created a feature which could fit. Please take a look at this preview: https://deploy-preview-1409--dtc-docs-preview.netlify.app/020_tutorial/030_generatehtml

In the right column, you should be able to see some issues regarding the current page.

The PR can be found here: https://github.com/docToolchain/docToolchain/pull/1409
